MoviesApp/Controller/BaseViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ViewController.swift
+//  MoviesApp
+//
+//  Created by claudia.apostol on 10/31/22.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+
+    let movieList = MoviesListProvider()
+    var filteredMovies: [Movie] = []
+    var sortCriteria: SortCriteria = .popularityDesc
```

Comment:
how about having the criteria properties be read-only (and overridable) so that in Favorites you don't have to override viewDidLoad, only the two properties?